scripts/gen-edpm-kustomize.sh

```diff
@@ -85,23 +85,25 @@ patches:
       path: /spec/nodes/edpm-compute-0/node/ansibleVars
       value: |
         ctlplane_ip: ${EDPM_COMPUTE_IP}
-        internal_api_ip: 172.17.0.100
-        storage_ip: 172.18.0.100
-        tenant_ip: 172.19.0.100
-        fqdn_internal_api: '{{ ansible_fqdn }}'
-    - op: replace
-      path: /spec/nodes/edpm-compute-0/node/ansibleSSHPrivateKeySecret
-      value: ${EDPM_ANSIBLE_SECRET}
```

Comment:
why removing the SSH secret?

Reply:
We don't need to duplicate for the nodes when provided for the https://github.com/openstack-k8s-operators/install_yamls/blob/main/scripts/gen-edpm-kustomize.sh#L193